robottelo/utils/issue_handlers/jira.py

```diff
@@ -169,12 +195,34 @@ def collect_dupes(jira, collected_data, cached_data=None):  # pragma: no cover
     stop=stop_after_attempt(4),  # Retry 3 times before raising
     wait=wait_fixed(20),  # Wait seconds between retries
 )
-def get_data_jira(issue_ids, cached_data=None):  # pragma: no cover
+def get_jira(jql, fields=None):
+    """Accepts the jql to retrieve the data from Jira for the given fields
+
+    Arguments:
+        jql {str} -- The query for retrieving the issue(s) details from jira
+        fields {list} -- The custom fields in query to retrieve the data for
+
+    Returns: Jira object of response after status check
+    """
+    params = {"jql": jql}
+    if fields:
+        params.update({"fields": ",".join(fields)})
+    response = requests.get(
+        f"{settings.jira.url}/rest/api/latest/search/",
+        params=params,
+        headers={"Authorization": f"Bearer {settings.jira.api_key}"},
+    )
+    response.raise_for_status()
+    return response
+
+
+def get_data_jira(issue_ids, cached_data=None, jira_fields=None):  # pragma: no cover
```

Comment:
```suggestion
def get_data_jira(issue_ids, cached_data=None, jira_fields=common_jira_fields):  # pragma: no cover
```
Wouldn't it be better to use `common_jira_fields`? It'll avoid getting a ton of unnecessary data.

Reply:
Handled within the code instead of arguments to param as its not a good practice.